src/sw.js

```diff
@@ -1,39 +1,42 @@
-var window = {};
-
-self.addEventListener('install', function (event) {
-  // Put `offline.html` page into cache
-  var offlineRequest = new Request('offline.html');
-  event.waitUntil(
-    fetch(offlineRequest).then(function (response) {
-      return caches.open('offline').then(function (cache) {
-        console.log('[oninstall] Cached offline page', response.url);
-        return cache.put(offlineRequest, response);
-      });
-    })
-  );
+const CACHE = 'cache-and-update';
```

Comment:
Rename file. I know, that sw equals serviceWorker, but we should use full sentences not only first letters or magic variables :P